src/components/editor-panels.js

```diff
@@ -2,30 +2,40 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Spinner, CheckboxControl, PanelBody, PanelRow } from '@wordpress/components';
+import { BaseControl, CheckboxControl, PanelBody, Spinner } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 
-const CheckboxesGroup = ( { options, values, onChange } ) => {
+const CheckboxesGroup = ( { options, values, onChange, defaultRequiredSlug } ) => {
```

Comment:
Nit: offer a default `null` value to show that this prop isn't required

```suggestion
const CheckboxesGroup = ( { options, values, onChange, defaultRequiredSlug = null } ) => {
```